great_ape_safe/ape_api/balancer.py

```diff
@@ -274,7 +290,7 @@ def _deposit_and_stake(self, pool, request, stake=True, destination=None):
     def stake(self, pool, mantissa, destination=None, dusty=False):
         pool_id = pool.getPoolId()
         destination = self.safe if not destination else destination
-        gauge_address = self.gauge_factory.getPoolGauge(pool)
+        gauge_address = self.get_preferential_gauge(pool.lower())
```

Comment:
I guess we should change all other references to `gauge_factory.getPoolGauge` as well

Reply:
resolved in f2f4c7cb07bf0477806711c775db4982393a4a5e